From 25a64e4e7ef6da605a86ec1bff18d2c3c6ed5329 Mon Sep 17 00:00:00 2001
Message-Id: <25a64e4e7ef6da605a86ec1bff18d2c3c6ed5329.1640255520.git.geliang.tang@suse.com>
From: Daniel Wagner <dwagner@suse.de>
Date: Thu, 1 Apr 2021 11:54:11 +0200
Subject: [PATCH] nvme: remove superfluous else in nvme_ctrl_loss_tmo_store
Git-commit: 25a64e4e7ef6da605a86ec1bff18d2c3c6ed5329
Patch-mainline: v5.13-rc1
References: bsn#51


If there is an error we will leave the function early. So there
is no need for an else. Remove it.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/nvme/host/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/nvme/host/core.c b/drivers/nvme/host/core.c
index b94a30e7298d..d2b4c5567209 100644
--- a/drivers/nvme/host/core.c
+++ b/drivers/nvme/host/core.c
@@ -3659,7 +3659,7 @@ static ssize_t nvme_ctrl_loss_tmo_store(struct device *dev,
 	if (err)
 		return -EINVAL;
 
-	else if (ctrl_loss_tmo < 0)
+	if (ctrl_loss_tmo < 0)
 		opts->max_reconnects = -1;
 	else
 		opts->max_reconnects = DIV_ROUND_UP(ctrl_loss_tmo,
-- 
2.31.1

